packages/shared/src/getTypegenOutput.ts

```diff
@@ -202,20 +202,13 @@ const collectInternalEvents = (lineArrays: { events: string[] }[][]) => {
 const displayEventsCausing = (lines: { name: string; events: string[] }[]) => {
   return lines
     .map((line) => {
-      return `${JSON.stringify(line.name)}: ${
-        unique(
-          line.events.map((event) => {
-            return event;
-          })
-        )
-          .map((event) => JSON.stringify(event))
-          .join(" | ") ||
-        /**
-         * If no transitions go to this guard/service/action, it's guaranteed
-         * to be caused by xstate.init.
-         */
```

Comment:
I believe that everything should already be handled upfront and there is no need to have this fallback here - it could only obscure some bugs for us.

Reply:
Actually - thanks to the test in the CLI, I've realized that sometimes things might actually be unreachable. But in that case, it's more correct to default to `never` here instead of `'xstate.init'`